Libraries/Mvc/ResultHelpers.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+
+namespace Lombiq.HelpfulLibraries.Libraries.Mvc
+{
+    public static class ResultHelpers
+    {
+        /// <summary>
+        /// Compresses the <paramref name="files"/> into a zip archive ready to be served by a controller.
+        /// </summary>
+        /// <param name="files">The file collection where key is the zip entry file name and value is the data.</param>
+        /// <param name="zipFileName">The file name of the archive without the ".zip" at the end.</param>
+        public static FileResult ZipFile(IDictionary<string, byte[]> files, string zipFileName)

```

Comment:
Azt hiszem láttam már többször azt, hogy ilyenkor a változó neve is jelzi, hogy "WithoutExtensions". Vagy egyszerűen csak csekkolod, hogy van-e a végén .zip (ignor case) és ha van, akkor nem rakod hozzá a végén.

Reply:
Jó ötlet, akkor legyen az utóbbi.